zerver/views/user_groups.py

```diff
@@ -96,8 +96,14 @@ def add_user_group(
 
 @require_member_or_admin
 @has_request_variables
-def get_user_group(request: HttpRequest, user_profile: UserProfile) -> HttpResponse:
-    user_groups = user_groups_in_realm_serialized(user_profile.realm)
+def get_user_group(
+    request: HttpRequest,
+    user_profile: UserProfile,
+    allow_deactivated: bool = REQ(json_validator=check_bool, default=False),
```

Comment:
Not sure whether this is required or not, but just added for now. Also, do we want to allow deactivated groups in queries like "get members of a group", "get subgroups of a group", etc.

Reply:
Probably a good question for the API design channel; I'm not sure.

Reply:
I think probably this change is definitely good; but you can ask the other ones for a follow-up. 

Also I'm going to rename this `get_user_group` function, I was super confused before realizing its name should be plural.